Spring.FluentContext.Web/FluentWebApplicationContext.cs

```diff
@@ -0,0 +1,79 @@
+using System;
+using Spring.Context;
+using Spring.Context.Support;
+using Spring.Objects.Factory.Config;
+using Spring.Objects.Factory.Support;
+
+namespace Spring.FluentContext.Web
+{
+    public class FluentWebApplicationContext : WebApplicationContext, IFluentConfigurableApplicationContext
```

Comment:
Please add tests to this class. There is a logic in `OnPreRefresh()` method and it should be covered with the tests proving it works as expected.